Lexplorer/Pages/NFTDetail.razor

```diff
@@ -84,11 +85,18 @@
                 <NFTContent nftMetadata="@nftMetadata" />
             </td>
         </tr>
-        @if (nft?.token != null)
+        @if ((nft?.token ?? nftMetadata?.JSONContent) != null)
         {
             <tr>
                 <td colspan="2">
-                    <MudButton Variant="Variant.Filled" Color="Color.Primary" OnClick="goToCollectionPage">View collection</MudButton>
+                    @if (nft?.token != null)
+                    {
+                        <MudButton Variant="Variant.Filled" Color="Color.Primary" OnClick="goToCollectionPage" Class="mr-2">View collection</MudButton>
+                    }
+                    @if (nftMetadata?.JSONContent != null)
+                    {
+                        <MudButton Variant="Variant.Filled" Color="Color.Primary" OnClick="showJSON" Class="mr-2">View JSON</MudButton>
```

Comment:
Could this be changed to "View Metadata JSON" or "View Metadata"? Just to be a bit more clearer?

Reply:
Sure, I went with the latter.